projects/packages/backup/changelog/update-backup-storage-ui-to-use-custom-retention-period-if-set

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Maybe it could be a patch

Reply:
Hmm, yes was thinking of same, will change it to patch